Library/Formula/eg.rb

```diff
@@ -0,0 +1,92 @@
+class Eg < Formula
+  desc "Expert Guide. Norton Guide Reader For GNU/Linux"
+  homepage "http://www.davep.org/norton-guides/"
+  url "http://www.davep.org/norton-guides/eg-1.00.tar.gz"
+  sha256 "e985b2abd160c5f65bea661de800f0a83f0bfbaca54e5cbdc2e738dfbbdb164e"
+  head "https://github.com/davep/eg.git"
+
+  depends_on "s-lang"
+
+  # clang complains about escaped newlines used in the source
+  # to form a multiline string. The patch converts said string
+  # to a more common C syntax.
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!